app/helpers/podcasts_helper.rb

```diff
@@ -42,6 +42,14 @@ def episode_description(episode, feed)
     description
   end
 
+  def episode_guid(episode, feed)
+    if feed.unique_guids?
+      "#{episode.item_guid}_#{feed.slug || feed.id}"
```

Comment:
This does mean... if the feed slug changes, all the <guid>s in that feed change.

But hopefully that wouldn't happen IRL?  Changing the slug would also change your feed's url.  Which would already be breaking things for listeners.

Reply:
switched to the immutable feed.id